src/redux/redux-store.js

```diff
@@ -1,22 +1,23 @@
-import {combineReducers, createStore, applyMiddleware} from "redux";
-import profileReducer from "./profile-reducer";
-import dialogsReducer from "./dialogs-reducer";
-import sidebarReducer from "./sidebar-reducer";
-import usersReducer from "./user-reducer";
-import authReducer from "./auth-reducer";
+import { combineReducers, createStore, applyMiddleware } from "redux";
+
+import profileReducer from "./profile-reducer/profile-reducer";
+import dialogsReducer from "./dialogs-reducer/dialogs-reducer";
+import sidebarReducer from "./sidebar-reducer/sidebar-reducer";
+import usersReducer from "./user-reducer/user-reducer";
+import authReducer from "./auth-reducer/auth-reducer";
 import thunkMiddleware from "redux-thunk";
-import {reducer as formReducer} from 'redux-form'
-import appReducer from "./app-reducer";
+import { reducer as formReducer } from 'redux-form'
+import appReducer from "./app-reducer/app-reducer";
 
 
 let reducers = combineReducers({
-    profilePage: profileReducer,
-    dialogsPage: dialogsReducer,
-    sidebar: sidebarReducer,
-    usersPage: usersReducer,
-    auth: authReducer,
-    form: formReducer,
-    app: appReducer
+	profilePage: profileReducer,
+	dialogsPage: dialogsReducer,
+	sidebar: sidebarReducer,
+	usersPage: usersReducer,
+	auth: authReducer,
+	form: formReducer,
+	app: appReducer
 });
 
 

```

Comment:
rule:
if you are in a directory called "aloha/" don't reuse that name in your file names,
in your case:
src/redux/redux-store.js -> becomes ->  src/redux/store.js